interactions/bot.py

```diff
@@ -102,7 +102,7 @@ def __init__(
     @property
     def guilds(self) -> List[Guild]:
         """Returns a list of guilds the bot is in."""
-        return [Guild(**_, _client=self._http) for _ in self._http.cache.self_guilds.view]
+        return [Guild(**_) for _ in self._http.cache.self_guilds.view]
```

Comment:
How come we're ditching the client? What if bot developers want to call methods in a guild from here?

Reply:
https://github.com/interactions-py/library/pull/670#discussion_r839827602